types/src/dkg/real_dkg/mod.rs

```diff
@@ -166,15 +205,44 @@ impl DKGTrait for RealDKG {
             &aux,
             &Player { id: my_index },
             rng,
-        )
+        );
+        // transcript for fast path
+        let fast_wtrx = pub_params
+            .pvss_config
+            .fast_wconfig
+            .as_ref()
+            .map(|fast_wconfig| {
+                WTrx::deal(
+                    fast_wconfig,
+                    &pub_params.pvss_config.pp,
+                    sk,
+                    &pub_params.pvss_config.eks,
+                    input_secret,
+                    &aux,
+                    &Player { id: my_index },
+                    rng,
+                )
+            });
+        Transcripts {
+            main: wtrx,
+            fast: fast_wtrx,
+        }
     }
 
     fn verify_transcript(
         params: &Self::PublicParams,
         trx: &Self::Transcript,
     ) -> anyhow::Result<()> {
+        // Verify fast path is present if and only if fast_wconfig is present.
+        assert_eq!(
```

Comment:
this is invoked in execution, potentially for a txn from a malicious validator. should not panic?